src/main/assembly/dist/res/localrun.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+USER_HOST="..." # to be filled in by the developer; this is the 'user@host' for Vagrant
+PRIVATE_KEY="..." # to be filled in by the developer; this is the private key for ssh-ing into Vagrant
```

Comment:
@janvanmansum would there be a better way to fill in these blanks? I mean, when developers use this project they need to fill in these values without any reference to dtap or something...